src/StoreKeeper/WooCommerce/B2C/Core.php

```diff
@@ -159,6 +160,8 @@ public function __construct()
             $this->loader->add_filter('wp_get_attachment_image_src', $media, 'getAttachmentImageSource', 999, 4);
             $this->loader->add_filter('wp_calculate_image_srcset', $media, 'calculateImageSrcSet', 999, 5);
         }
+        $this->addCategorySeoFields();
+        $this->addProductSeoFields();
```

Comment:
I can see you synchronize the data. But where do you show it on the product and category pages in frontend?

Reply:
It's shown when the hooks are called on the i.e. `product_cat_edit_form_fields` and `edit_form_advanced`